dub.sdl

```diff
@@ -22,13 +22,13 @@ configuration "library" {
 }
 
 configuration "library-nonet" {
-	dependency "vibe-d:http" version=">=0.7.30 <=0.9.0" optional=true
+	dependency "vibe-d:http" version=">=0.7.31 <=0.9.0" optional=true
```

Comment:
This is probably not really necessary, as DUB only uses very basic functionality (it should be compatible with very old versions actually). The interesting place is just in dub.selections.json.